### PR TITLE
fetched data for homepage categories & product listing page

### PR DESCRIPTION
**Pull request by devsurajrai**
Fetched data for homepage categories & product listing page. Please review the PR

Screenshot 2022-03-22 at 1 51 20 PM
.

### CODE REVIEW

File: src/components/ProductCard.component/product-card.css [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
 .card-badge {
     /* position  */
```
Comment (palash23jain) [inline src/components/ProductCard.component/product-card.css:3]: this comment can be removed

**Review by palash23jain — APPROVED** [review]
Overall good structure maintained, good coding standards

File: src/components/HomeMain.component/Categories.component/HomeCategories.jsx [hunk 2]
```diff
@@ -1,14 +1,17 @@
...
-const HomeCategories = () => {
+const HomeCategories = ({ category }) => {
   return (
     <div className="cat-card br-sm">
-      <img className="res-img" src={racket} alt="" />
-      <a className="button btn-sm cat-card-btn text-sm" href="">
-        RACQUETS
-      </a>
+      <img className="res-img" src={category.categoryImgUrl} alt="" />
+      <Link
```
Comment (sushmita12345) [inline src/components/HomeMain.component/Categories.component/HomeCategories.jsx:9]: Suggestion : fill alt attribute

**Review by sushmita12345 — APPROVED** [review]
Code looks good, you can go ahead and merge.

**Review by varnita619 — APPROVED** [review]
Good naming convension and folder structure

File: src/components/ProductCard.component/product-card.css (hunk 1 above)
Comment (devsurajrai) [reply src/components/ProductCard.component/product-card.css:3]: Thanks ! Will do that.

File: src/components/HomeMain.component/Categories.component/HomeCategories.jsx (hunk 2 above)
Comment (devsurajrai) [reply src/components/HomeMain.component/Categories.component/HomeCategories.jsx:9]: will do that.

**Pull request merged**